test-integ/peering_commontopo/ac3_service_defaults_upstream_test.go

```diff
@@ -0,0 +1,266 @@
+package peering
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/consul/testing/deployer/topology"
+	"github.com/hashicorp/go-cleanhttp"
+	"github.com/itchyny/gojq"
+	"github.com/stretchr/testify/require"
+
+	"github.com/hashicorp/consul/api"
+	"github.com/hashicorp/consul/sdk/testutil/retry"
+	libassert "github.com/hashicorp/consul/test/integration/consul-container/libs/assert"
+)
+
+var ac3SvcDefaultsSuites []sharedTopoSuite = []sharedTopoSuite{
+	&ac3SvcDefaultsSuite{DC: "dc1", Peer: "dc2"},
+	&ac3SvcDefaultsSuite{DC: "dc2", Peer: "dc1"},
+}
+
+func TestAC3SvcDefaults(t *testing.T) {
+	runShareableSuites(t, ac3SvcDefaultsSuites)
+}
+
+type ac3SvcDefaultsSuite struct {
+	// inputs
+	DC   string
+	Peer string
+
+	// test points
+	sidServer  topology.ServiceID
+	nodeServer topology.NodeID
+	sidClient  topology.ServiceID
+	nodeClient topology.NodeID
+
+	upstream *topology.Upstream
+}
+
+var _ sharedTopoSuite = (*ac3SvcDefaultsSuite)(nil)
```

Comment:
I don't think this is necessary as `ac3SvcDefaultsSuite` struct implements the `sharedTopoSuite` interface